### DISCUSSION TITLE
How to set the semanticCommitType for updates to GitHub Action Workflows

### DISCUSSION THREAD

**Discussion by gr2m** [renovatebot/renovate#8385] [Request Help] [answered]
Example: https://github.com/octokit/types.ts/pull/240

Here is my current configuration:
https://github.com/octokit/.github/blob/b3cfdba7f34e89b6a959524cbaff790a861e0f49/renovate.json

Is there a `depTypeList` value for GitHub Action workflows?

**Comment by viceice** [comment, marked as answer]
sure, it's `action`. see sample:

https://github.com/renovatebot/presets-internal/blob/7da2897616ad01c9726e5ce2020f57ba64de89a9/base.json#L55-L59

**Reply by viceice** [reply]
The linked package rule above will auto pin the full action version when available 🙃
